sql/mob_spawn_points.sql

```diff
@@ -70634,5 +70634,3 @@ INSERT INTO `mob_spawn_points` VALUES (17977406,'Darkness','Darkness','12987','0
 -- ------------------------------------------------------------
 -- Dynamis-Jeuno [D] (Zone 297)
 -- ------------------------------------------------------------
-
```

Comment:
didn't need to trim that, its unrelated to the NM you are adding

Reply:
i actually had never touched that file, unsure why it said i made a change to it (among others)